I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 88, in script
    init python:
  File "game/script.rpy", line 111, in <module>
    if hasattr(config, "hyperlink_focus") and hasattr(config, "hyperlink_leave"):
Exception: config.hyperlink_leave is not a known configuration variable.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 88, in script
    init python:
  File "E:\renpy\renpy\ast.py", line 834, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "E:\renpy\renpy\python.py", line 1187, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 111, in <module>
    if hasattr(config, "hyperlink_focus") and hasattr(config, "hyperlink_leave"):
  File "E:\renpy\renpy\defaultstore.py", line 93, in __getattr__
    raise Exception('config.%s is not a known configuration variable.' % (name))
Exception: config.hyperlink_leave is not a known configuration variable.

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
novel 1.0
Tue Aug 12 09:21:43 2025
